rewrite-java-11/src/main/java/org/openrewrite/java/Java11JavadocVisitor.java

```diff
@@ -100,6 +100,10 @@ private void init() {
         int i = 3;
         for (; i < sourceArr.length; i++) {
             char c = sourceArr[i];
+            if (c == '\r') {
```

Comment:
Temporary, unless peers are okay with the abstraction.
Note: This does not preserve a mix of LF and CRLF. This prevents edge case formatting issues that are created when substrings and new LineBreaks are generated that only check the previous character. Also, relies on the normalization of new lines.